amd_openvx_extensions/amd_rpp/CMakeLists.txt

```diff
@@ -125,52 +131,53 @@ list(APPEND SOURCES
         source/internal_publishKernels.cpp
     )
 
-link_directories(${AMDRPP_LIBRARIES_DIR})
-
 if(GPU_SUPPORT AND "${BACKEND}" STREQUAL "OPENCL"  AND OpenCL_FOUND)
-    message("-- ${Green}amd_rpp -- Building with OpenCL${ColourReset}")
+    message("-- ${Green}vx_rpp -- Building with OpenCL${ColourReset}")
     set(ENABLE_OPENCL 1)
     set(ENABLE_HIP 0)
     add_definitions(-DENABLE_OPENCL=${ENABLE_OPENCL} -DENABLE_HIP=${ENABLE_HIP} -DRPP_BACKEND_OPENCL=1)
     include_directories(${OpenCL_INCLUDE_DIRS} ${OpenCL_INCLUDE_DIRS}/Headers)
+    set(LINK_LIBRARY_LIST ${LINK_LIBRARY_LIST} ${OpenCL_LIBRARIES})
     add_library(vx_rpp SHARED ${SOURCES})
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DGPU_SUPPORT=1 -DRPP_BACKEND_OPENCL=1") # Currently used by rpp. To be removed.
-    target_link_libraries(vx_rpp amd_rpp boost_filesystem boost_system ${OpenCL_LIBRARIES})
 elseif (GPU_SUPPORT AND "${BACKEND}" STREQUAL "HIP" AND HIP_FOUND)
-    message("-- ${Green}amd_rpp -- Building with HIP${ColourReset}")
+    message("-- ${Green}vx_rpp -- Building with HIP${ColourReset}")
     set(ENABLE_OPENCL 0)
     set(ENABLE_HIP 1)
     add_definitions(-DENABLE_OPENCL=${ENABLE_OPENCL} -DENABLE_HIP=${ENABLE_HIP} -DRPP_BACKEND_HIP=1 -D__HIP_PLATFORM_AMD__)
     include_directories(${ROCM_PATH}/include)
     link_directories(${HIP_PATH}/lib)
+    set(LINK_LIBRARY_LIST ${LINK_LIBRARY_LIST} hip::host)
     add_library(vx_rpp SHARED ${SOURCES})
     set_target_properties(openvx PROPERTIES LINKER_LANGUAGE CXX)
     set_target_properties(openvx PROPERTIES POSITION_INDEPENDENT_CODE ON)
     # TBD -- CHECK to FIX
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DGPU_SUPPORT=1 -DRPP_BACKEND_HIP=1") # Currently used by rpp. To be removed.
-    target_link_libraries(vx_rpp amd_rpp boost_filesystem boost_system hip::host)
```

Comment:
@LakshmiKumar23 had boost required but never checked for library, had to fix in this pr.

Reply:
> @kiritigowda The `mvdeploy_api.h` seems to included even if `BUILD_DEV=OFF`. Also seeing the mv_deploy scripts in model compiler
> 
> ```
> svcbuild@Legolas:~/test/MIVisionX/build-nodev/mivisionx-2.4.0-Linux/opt/rocm/include/mivisionx$ ls 
> mvdeploy_api.h
> svcbuild@Legolas:~/test/MIVisionX/build-nodev/mivisionx-2.4.0-Linux/opt/rocm/libexec/mivisionx/model_compiler/mv_deploy$ ls
> mv_extras_postproc.cpp  mv_extras_postproc.h
> ```

@kiritigowda can you look into this also?